src/univariate/continuous/landau.jl

```diff
@@ -0,0 +1,191 @@
+using StaticArrays
```

Comment:
I assume dependencies should be loaded in `src/Distributions.jl`. I'm a bit surprised though that Distributions depends on StaticArrays but StaticArrays doesn't seem to be imported.

Reply:
I think it only uses them in test, so technically we should change that in Project.toml to reflect this

Reply:
Yes, I thought so as well.

Reply:
Ref: https://github.com/JuliaStats/Distributions.jl/pull/1246